test/unit/facet_test.rb

```diff
@@ -14,12 +14,6 @@ class FacetTest < ActiveSupport::TestCase
     Facets.stubs(:configuration).returns(@config)
   end
 
-  teardown do
```

Comment:
Without those lines, cloning `Host::Managed` will try to clone `test_facet`. I don't think it's desirable.

After this test Host::Managed will contain all properties related to `test_facet` and `namespaced_facet`